extension/src/test/CliSettingsLoader.test.ts

```diff
@@ -65,28 +65,31 @@ suite("CLI Settings Loader Tests", function () {
       errorMsg = (e as Error).message;
     }
 
-    let expectedErrMsg = "";
-    if (process.platform === "win32") {
-      expectedErrMsg = `ENOENT: no such file or directory, open '.vscode\\launch.json'`;
-    }
+    const expectedErrMsg = process.platform === "win32" ? getENOENT() : "";
     assert.deepStrictEqual(
       errorMsg,
       expectedErrMsg,
       "Unexpected error message"
     );
 
-    if (process.platform !== "win32") {
-      // Why is the properties not undefined on windows?
-      assert.deepStrictEqual(
-        launchSettings.server,
-        undefined,
-        "Expected 'server' property to be undefined"
-      );
-      assert.deepStrictEqual(
-        launchSettings.serverInstance,
-        undefined,
-        "Expected 'serverInstance' property to be undefined"
-      );
-    }
+    // if (process.platform !== "win32") {
+    //   // Why is the properties not undefined on windows?
+    //   assert.deepStrictEqual(
+    //     launchSettings.server,
+    //     "",
+    //     "Expected 'server' property to be undefined"
+    //   );
+    //   assert.deepStrictEqual(
+    //     launchSettings.serverInstance,
+    //     "undefined",
+    //     "Expected 'serverInstance' property to be undefined"
+    //   );
+    // }
```

Comment:
@jwikman apparently this behaves the same locally for me now. Not sure what to test for, maybe remove completely? The `launchSettings` variable keeps the values and I guess that's expected but not something we'd like to test for.

Reply:
I'm not really following here... :)

Anything I can try out on my windows machine?

Reply:
We'll get back to this later, if needed.

Reply:
Before the launchsettings was:

Win32:
```
{
    server: "http://localhost"
    serverInstance: "BC666"
}
```

Linux:
```
{
    server: undefined
    serverInstance: undefined
}
``` 

Something must have been updated because locally (my computer) linux now seems to get the same result as Win32. However the container till gets the undefined linux result above. 

The question was intended to be more like: Is there any expected behavior we should test for in `launchSettings` after `CliSettingsLoader.getLaunchSettings("");` has failed on (line 63)? I can't really think of any. I first added the assertion after finding that the properties were undefined. But apparently their not. I don't mind leaving this in for now. We won't think twice about removing it later anyway :D

Reply:
Leave it for now